src/ouroboros/vlc_db/vlc_pose.py

```diff
@@ -33,3 +33,6 @@ def to_descriptor(self):
     @classmethod
     def from_descriptor(cls, d):
         return cls(time_ns=d[0], position=d[1:4], rotation=d[4:])
+
+    def as_matrix(self):
```

Comment:
(minor) Kinda annoying, but if we're using `spark_dataset_interfaces` elsewhere we could consider the pose implementation (given it's slightly more tested / has slightly more features)